dnf/base.py

```diff
@@ -1970,7 +1970,6 @@ def _upgrade_internal(self, query, obsoletes, reponame, pkg_spec=None):
             q = q.union(obsoletes)
         # provide only available packages to solver otherwise selection of available
         # possibilities will be ignored
```

Comment:
shouldn't the comment be changed as well?

Reply:
Good point, corrected